### ISSUE TITLE
v5 API support

### ISSUE THREAD

**Issue by guedressel** [Gandi/letsencrypt-gandi#37]
Since I'm a new customer of gandi.net my account seems not to be available in the v4.gandi.net infrastructure (including the "old" v4 RPC API).
As far as I read in the web about this there was/is/will be a big change in the API moving it to REST for the v5 infrastructure.

Since I don't have a v4 API-KEY and I'm not able to get one, I'm eventually not able to use this certbot plugin.

```
Fault: <Fault 510150: 'Error on object : OBJECT_ACCOUNT (CAUSE_NORIGHT) [Invalid API key]'>
```

**Comment by webcracy** [comment]
Indeed. For the moment you'll need to use the v5 website to create free certificates and manage your instance. I still encourage you to contact the customer care team and share your need and use case, they might be able to help out.

**Comment by bcien** [comment]
Maybe the support team can manually create an API key for you. They are very helpful, so if its an option, they will probably do it.
Please post your experience here to know if it is an option for new users. 

Anyways, a new version of this plugin compatible with the new API will eventually need to be implemented...

**Comment by baloo** [comment]
well, we have an automated ssl setup in v5, making this plugin unnecessary. (Go directly in your admin, enable SSL, done)
